tests/queries/0_stateless/01301_aggregate_state_exception_memory_leak.sh

```diff
@@ -16,5 +16,5 @@ for _ in {1..1000}; do
     if [[ $elapsed -gt 30 ]]; then
         break
     fi
-done 2>&1 | grep -o -P 'Query memory limit exceeded' | sed -r -e 's/(.*):([a-Z ]*)([mM]emory limit exceeded)(.*)/\2\3/' | uniq
```

Comment:
Why did you change this? It doesn't seem to be connected with HDFS.

Reply:
it's not connected, yes. I just have an error seems related to different sed version:
`sed: -e expression #1, char 52: Invalid range end` it does not like [a-Z ]*
And I started to wonder what is the point of this sed|uniq commands in the pipe? Aren't they pointless?

Reply:
thanks for the review btw,
I planned to ask someone about this line;)

Reply:
> Aren't they pointless?

At this current form it is pointless. If you check the history of the file, you can see that it was not pointless. I will approve.

Goede gedaan, meneer!

Reply:
Köszi, haver!